vignettes/miaSim.Rmd

```diff
@@ -36,14 +36,9 @@ knitr::opts_chunk$set(cache = FALSE,
                         warning = FALSE)
 ```
 
-
 # Introduction
 
-`miaSim` implements tools for microbiome data simulation based on
-different ecological modeling assumptions. These can be used to
-simulate species abundance matrices, including time series. For a detailed function documentation, see the
-[function reference page](https://microbiome.github.io/miaSim/reference/index.html)
-
+`miaSim` implements tools for microbiome data simulation based on different ecological modeling assumptions. These can be used to simulate species abundance matrices, including time series. For a detailed function documentation, see the [function reference page](https://microbiome.github.io/miaSim/reference/index.html)
```

Comment:
That homepage is outdated, we already have an issue open to fix that and automated homepage builds.